modules/virt-about-static-and-dynamic-ssh-keys.adoc

```diff
@@ -34,7 +34,7 @@ You can add the key by using one of the following methods:
 
 You can enable dynamic SSH key management for a VM with {op-system-base-full} 9 installed. Afterwards, you can update the key during runtime. The key is added by the QEMU guest agent, which is installed with Red Hat boot sources.
 
-You can disable dynamic key management for security reasons. Then, the VM inherits the key management setting of the image from which it was created.
+If the image used for the VM is set to use dynamic SSH key management, the VM uses the feature by default. If not, you can enable it by updating the configuration of the VM.
```

Comment:
Can it be rearranged a bit please? Currently it is a bit misleading. It is telling me, if it is not present in the image I can enable it on the vm. But I believe the goal is to communicate, that when it is not set on the vm, the ability to use it would be dependent on associated image settings.

Reply:
@dbasunag , thank you for the review. I did try to communicate the same thing (in a more UX-focued manner), but I think misunderstood the difference between the key management setting on the image as opposed to the dynamic key management set on the VM. How about a rewrite like this, then?

"When dynamic key management is disabled, the default key management setting of a VM is determined by the image used for the VM."

Reply:
Sounds good.